src/pages/i-need-fuel.js

```diff
@@ -12,6 +14,32 @@ import { selectSession } from '~/store/selectors'
 
 const { publicRuntimeConfig } = getConfig()
 const { irc: ircURLs } = publicRuntimeConfig
+const unsupportedPlatforms = {
+  Sony: {
+    type: 'error',
+    message: `
+      The built-in PS4 browser is currently not supported.
+      This is due to bugs in the PS4 browser, and outside of our control.
+      Please use your phone or computer instead.
+    `,
+  },
+  Xbox: {
+    type: 'warn',
+    message: `
+      It appears you're connecting from a mobile device.
+      Ensure your device's screen and browser remains active through the course of your rescue.
+      You may loose connection otherwise!
+    `,
+  },
+  Generic: {
+    type: 'warn',
+    message: `
+      It appears you're connecting from a mobile device.
+      Ensure your device's screen and browser remains active through the course of your rescue.
+      You may loose connection otherwise!
```

Comment:
"Lose connection", not loose. Also "throughout your rescue" is probably better language.

Reply:
This is why I shouldn't write things while tired AF